trunk/SUAVE/Optimization/Package_Setups/scipy_setup.py

```diff
@@ -54,14 +54,24 @@ def SciPy_Solve(problem,solver='SLSQP', sense_step = 1.4901161193847656e-08): #
     
     x   = ini/scl
     bnds = np.zeros((len(inp),2))
+    lb   = np.zeros(len(inp))
```

Comment:
Possibly move these definitions and calculations into the wrapper control flow so that they aren't executed when the solver being used doesn't require them. Also recommend vectorizing the code for speed-up unless there's something about the data structure I'm not seeing that prevents that.

Reply:
I kept it here because did not want to create an additional loop with one line. There is no wrapper for differential evolution/particle swarm optimization outside of Nexus.